oak_tensorflow_service/src/tflite.rs

```diff
@@ -80,18 +80,27 @@ impl TfliteModel {
         let input_bytes_len = input_bytes
             .len()
             .try_into()
-            .map_err(|error| anyhow!("Failed to convert usize to u64: {}", error))?;
-        // TODO(#3297): Allocate enough bytes for the TensorFlow Lite model.
-        let mut output_bytes = vec![0; input_bytes.len()];
-        let mut output_bytes_len = 0u64;
+            .map_err(|error| anyhow!("Failed to convert input bytes length to u64: {}", error))?;
+
+        // TensorFlow Lite is the owner of the corresponding `output_bytes` buffer.
```

Comment:
I don't think this is the best design. AFAICT the current implementation leaks the output bytes. I think the buffer should be owned by the Rust code and initialised to a maximum size. If less is used by TF lite it can return a smaller size so that the buffer can be shrunk.

Reply:
Current problem is that the Rust code doesn't know in advance what will be the output size for a certain model.

Another solution could be to have an output size as a parameter for the `Init(tf_model, output_size)` IDL call, that initializes the runtime.

Reply:
Is it not possible to calculate the output size from the model in TF Lite? In that case the output size could be returned by the TF Lite wrapper either as a result from Init, or as a separate function.